File: src/test/java/info/kfgodel/dyna/testobjects/GlassesWearer.java
package info.kfgodel.dyna.testobjects;

/**
 * This interface is created for testing superman / kent objects without repeating code
 * Date: 07/05/19 - 00:11
 */
public interface GlassesWearer {

  String HAS_GLASSES_PROPERTY= "hasGlasses";

  boolean getHasGlasses();
  void setHasGlasses(boolean newState);
}

File: src/test/kotlin/info/kfgodel/dyna/objects/GlassesWearer.kt
package info.kfgodel.dyna.objects

/**
 * This interface is created for testing superman / kent objects without repeating code
 * Date: 07/05/19 - 00:11
 */
interface GlassesWearer {
  var hasGlasses: Boolean

  companion object {
    const val HAS_GLASSES_PROPERTY = "hasGlasses"
  }
}
